src/fh-family-experience-web/Pages/FamilyHub.cshtml.cs

```diff
@@ -53,11 +59,16 @@ public async Task<IActionResult> OnPostAsync()
         }
 
         TempData["PostCode"] = Postcode;
-        PostcodeIOResponse = await _postcodeLookupService.GetPostcodeAsync(Postcode);
+        PostcodeIOResponse = await _postcodeLookupService.GetPostcodeAsync(Postcode!);
+
+        if (PostcodeIOResponse is null)
+            throw new Exception("Failed to get response from Service Directory API");
```

Comment:
Ideally we shouldn't be throwing `Exception` directly.
I believe Sonar will flag this, which we'll add once it supports .net 6.